src/Stream.php

```diff
@@ -357,4 +362,15 @@ private function setStream($stream, string $mode = 'r') : void
 
         $this->resource = $resource;
     }
+
+    /**
+     * Determine if a resource is one of the resource types allowed to instantiate a Stream
+     *
+     * @param resource $resource Stream resource.
+     * @return bool
+     */
+    protected function isAllowedStreamResourceType($resource)
+    {
+        return in_array(get_resource_type($resource), self::ALLOWED_TYPES);
```

Comment:
may need strict check : 

```suggestion
        return in_array(get_resource_type($resource), self::ALLOWED_TYPES, true);
```

Reply:
Thanks for the feedback.

Reply:
you seems push to wrong repo branch, should push to your fork branch instead of this repo branch

Reply:
please push to your fork ( as this PR ):

https://github.com/settermjd/laminas-diactoros/tree/expand-list-of-resource-types

instead of this repo branch:

https://github.com/laminas/laminas-diactoros/tree/expand-list-of-resource-types

Reply:
I must have done that by accident. Sorry.

Reply:
That said, I updated the PR to include strict type checking.